### PR TITLE
Adding support to cross-compile for CoreRT. extending dir.targets to

### PR DESCRIPTION
**Pull request by SedarG**
handle CoreRT targeting packs, and dir.props to define
PackageTargetFramework and PakcageTargetRuntime for netcoreapp1.2 and
netcoreapp1.2corert TargetGroups

@ericstj, @weshaggard, @jkotas PTAL
/cc @dotnet/corert-contrib

### CODE REVIEW

File: dir.targets [hunk 1]
```diff
@@ -93,4 +93,76 @@
     </ItemGroup>
   </Target>
 
+  <!-- Support for CoreRT Targeting Packs
+       CoreRT Targeting requires differentiating the targetting pack according to the platforms.
```
Comment (joshfree) [inline dir.targets]: nit: targetting -> targeting

**Comment by joshfree** [comment]
Thanks @SedarG 👍

File: dir.targets [hunk 2]
```diff
@@ -93,4 +93,76 @@
...
+       i.e.: the targetting pack for linux is different than that of windows. In order to support this
```
Comment (jkotas) [inline dir.targets]: If you do the split targeting pack, it is hard to tell where to stop. Do you also need to split accross CPU architectures? The implementation corelib is different for each x86, x64, arm, arm64. 

I am not saying that it is a good idea to split it accross CPU architectures. I am just pointing out that this plan is not correct by construction, and just makes CoreRT different from CoreCLR.

Comment (SedarG) [inline dir.targets]: Looks like there was a misunderstanding. I thought @jkotas were stating that due to differences in System.Private.Interop we would need different targeting pack for each platform. And that's why I designed the production and consumption of the targeting packs of CoreRT in this way. Apparently, the differences in System.Private.Interop won't necessarily be effective in consumption; we might not need to reference/consume those platform specific APIs  from CoreFX libraries, in which case using Windows x64 assets for all platforms would be fine. 

I'll close this PR and will prep another one according to that plan.

**Pull request closed without merging**